.github/workflows/c-patmos-tests.yml

```diff
@@ -43,12 +43,20 @@ jobs:
           path: core/src/main/resources/lib/c/reactor-c
           ref: ${{ inputs.runtime-ref }}
         if: ${{ inputs.runtime-ref }}
-      - name: Run Patmos smoke tests
+      - name: Run Patmos tests
         run: |
           ./gradlew core:integrationTest \
             --tests org.lflang.tests.runtime.CPatmosTest.* \
             core:integrationTestCodeCoverageReport
           rm -rf test/C/src-gen
+      - name: Smoke tests
+        run: |
+          for file in $(find test/C/bin -type f -name '*Patmos*'); do
+            echo "Inspecting file: $file"
+            file "$file"
+            echo "Running pasim on $file"
+            pasim "$file" || echo "Error running pasim on $file"
```

Comment:
Would it not be better to override `getExecCommand` in `CPatmosTest`? That way the errors get caught in the testing framework rather than in a separate CI step. To make this work, you'd have to rename `buildPatmosUnthreadedTests` to `runPatmosUnthreadedTests` and change the test level accordingly.